src/layouts/default.vue

```diff
@@ -92,14 +87,20 @@ const embeddedPage = {
       isSidebarVisible,
       isSearchRoute,
       headerHasTwoRows,
-      mainContentRef,
-      mainRef,
     }
   },
 }
 export default embeddedPage
 </script>
+
 <style scoped>
+.sidebar {
+  height: calc(100vh - var(--header-height, 81px));
+}
+.has-sidebar .sidebar {
+  width: 325px;
```

Comment:
The sidebar width in the mockups is 316px. It keeps changing in the code, for some reason :)

Reply:
I've made this a global css var so it is consistent everywhere!